renku-cli-model/src/test/scala/io/renku/cli/model/diffx/CliDiffInstances.scala

```diff
@@ -24,71 +24,77 @@ import io.renku.graph.model.diffx.ModelTinyTypesDiffInstances
 
 trait CliDiffInstances extends ModelTinyTypesDiffInstances {
 
-  implicit val cliPersonDiff: Diff[CliPerson] = Diff.derived[CliPerson]
+  implicit lazy val cliPersonDiff: Diff[CliPerson] = Diff.derived[CliPerson]
```

Comment:
I tried to get the order right: :-) but `lazy` is actually more convenient 👍🏼

Reply:
You caught me on naughty things :)